### PR TITLE
New version of CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by jlstpaul**
This is the rewrite of CONTRIBUTING.md (formerly in Google Doc).  Still need to complete the "creating good issues and pull requests" section.

### CODE REVIEW

**Comment by e-lo** [comment]
> Still need to complete the "creating good issues and pull requests" section.

I believe the best way to enforce this is through good issue and PR templates.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+PMT Group Member [GitHub Access](https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization): Admin
+ 
+### Contributors
+
+Contributors actively work to develop the TIDES specification. They propose additions, modifications, and improvements to the speciation document through issues and pull requests in this GitHub repository.
+
+Contributors must request [here][contributor-registration] to be registered in order to gain access.  Requests to become a Contributor must be approved by project Leadership.
+ 
+Contributor Group Members: 
+The list of registered contributors is maintained in the [contributors.md][contributors-file] file.
```
Comment (botanize) [inline CONTRIBUTING.md:99]: This is an invalid link, should be something like, `[contributors.md](CONTRIBUTORS.md)`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+Leadership will convene a governance group to refine this and decide the approval process for Version 1.0 and the governance and approval process for future revisions to the specification.
+ 
+## [Code of Conduct](code-of-conduct)
+ 
+Contributors to the TIDES Project are expected to read and follow the Code of Conduct for the project in the [CODE_OF_CONDUCT.md][code-of-conduct-file] file.
+
+[contributor-registration]: https://forms.office.com/Pages/ResponsePage.aspx?id=i_a_3SpIc0WB4P74FWpP0Hpd6kyRp1VEg8rnx5-CwORUMFFGTzBYRktEMkJRWVg4Qlg3SkM0VEJKVi4u
+[license-file]: https://www.google.com
+[contributors-file]: https://www.google.com
+[code-of-conduct-file]: https://www.google.com
```
Comment (botanize) [inline CONTRIBUTING.md:126]: Replace these with correct links, e.g., `[license-file]: LICENSE`, `[contributors-file]: CONTRIBUTORS.md`, and `[code-of-conduct-file]: CODE_OF_CONDUCT.md`.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+Development of the TIDES specification shall be managed by the following groups:
+* Leadership
+* Product Management Team
+* Registered Contributors
+* All Other Stakeholders
+
+These groups will have the following rights and responsibilities:
+
+### Leadership
```
Comment (botanize) [inline CONTRIBUTING.md:65]: Make this linkable from CODE_OF_CONDUCT.md

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+* creation, scoping, and management of working groups, 
+* approval of the final specification for Version 1.0
+* approval of changes to project governance
+
+Leadership Group Members
+* John Levin, Metro Transit (Minneapolis-St. Paul, MN)
+
+Leadership Group Member [GitHub Access](https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization): Admin
+ 
+### Product Management Team (PMT)
```
Comment (botanize) [inline CONTRIBUTING.md:78]: Make this linkable from CODE_OF_CONDUCT.md

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+>  
+> c. The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.
+>  
+> d. I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.
+>  
+Attribution: This Contributor Agreement is adapted from the node.js project available here: https://github.com/nodejs/node/blob/main/CONTRIBUTING.md 
+
+## [How to Create Good Issues or Pull Request](howto)
+ 
+Content to be added.
```
Comment (botanize) [inline CONTRIBUTING.md:47]: Let's remove this, but add,
```md
## [How to write good commit messages](git-commit)
1. Separate subject from body with a blank line.
2. Limit the subject line to around 50 characters.
3. Capitalize the subject line.
4. Do not end the subject line with a period.
5. Use the imperative mood in the subject line, e.g., "fix schema file names", "define timestamp format".
6. Wrap the body at 72 characters.
7. Use the body to explain what and why.
8. [Reference issues or pull-requests on GitHub](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)
```

Comment (e-lo) [reply CONTRIBUTING.md:47]: Suggest just referencing: https://www.conventionalcommits.org/en/v1.0.0/#summary or similar

Comment (botanize) [reply CONTRIBUTING.md:47]: It's a little heavy handed, e.g., MUST use a prefix

Comment (jlstpaul) [reply CONTRIBUTING.md]: I have moved this best practices into the How to Commit section and removed the How to Create Good Issues and Pull Requests section.  If you want to make further edits to this section, please provide specific recommended text.

**Pull request merged**